packages/side-nav/src/CollapseToggle/CollapseToggle.tsx

```diff
@@ -33,6 +33,7 @@ export function CollapseToggle({
 
   return (
     <Tooltip
+      usePortal={false}
```

Comment:
Why? I might flag this as a potentially breaking change

Reply:
I added a few changsets, probably while you were reviewing. I added a few lines on this.